charters/DataPortal/charter.md

```diff
@@ -0,0 +1,69 @@
+# Data Portal
+
+
+## Description
+The Data Portal is the landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience (UX) for users to explore cellular resolution data.
+
+## Definitions
+*DCP* The Data Coordination Platform is the name given to the entire system used to ingest, validate, store, analyzes, and make available the data in the Human Cell Atlas project.
+
+*Data Browser* A DCP application which provides a faceted search capability.
+
+## Objectives
+The objective of the Data Portal is to provide the DCP website implementation. It provides the access point for users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis at petabyte scale on public clouds.
+
+## In-scope
+
+### Core capabilities
+* Implement a UX, visual, and branding design with input from the HCA brand working group.
+* Maintain a content management system that meets the needs of Data Portal content developers.
+* Integrate the Data Browser User Interface (UI) and Data Portal UI into a seamless UX and provide links to other DCP applications (e.g. the Ingestion Service UI).
+* Provide for frequent (minimum daily) updates of summary statistics (cells, organs, donors, projects, etc.) on the DCP home page.
```

Comment:
I'm unsure about the daily cadence here. My understanding of the portal was that it was designed to coordinate data releases to the community, whereas this suggests continuous release. Who is therefore responsible for the coordinated release of data to the community (i.e. ensuring all data is QC-ed to the same standard and all metadata upgraded to the same schema version)? Are you expecting the data browser / data store / query service will do this? Or do you really expect users of the data to do this?

Reply:
This is just referencing the cadence at which the Portal will update the summary statistics displayed on the Portal home page which is independent of the ingestion of the data. So the stats won't be immediately up to date with the data but may lag by the minimum. In the implementation I think it's more often than once per day, but that is a detail.

The portal does not "coordinate data releases to the community" - it is the UI that shows users what is there, The other services ingest data, create and index bundles, etc. The Data Browser (separate charter) displays data according to the indexes created, and the Data Browser is one of the applications accessed via the Data Portal. QC of data and metadata upgrade are outside the scope of the Data Portal and Data Browser charters.

Reply:
I think this comment is more related to the Browser than the Portal, but seeing as we started this thread I'll continue here...

Making it out of scope for the portal/browser is not unreasonable, but _someone_ has to coordinate releases. I would like to see impacted charters addressing the issue. The DCP will contain data that has been ingested over time, using different metadata standards, different analysis pipeline versions, different QC panels, etc. I'd expect the Data Portal / Browser (this charter is clearly the wrong place for this discussion following the split!) to either expect some sort of stamp that indicates the data is up to current standards, or else to provide this stamp themselves. Otherwise, you will face a considerable amount of engineering work to support display of all possible versions of every byte of data in the DSS, and pass the problem onto users when they download the resulting data. I have a suspicion this will be an enormous problem once imaging data starts to be ingested, for example.

Reply:
It is worth noting that this will very likely fall under the remit of the data operations team. That being said, just as the ingestion service, metadata schema and data store charters contain reciprocal statements about how responsibilities for different angles of metadata specification and implementation are divided. I would assume the data browser/portal will need to be linked to these efforts as they will create display requirements both for static and browsable content.

Reply:
Of note, the secondary analysis team thus far plans to reanalyze all data when a pipeline has a major version update.